### PR TITLE
travis initial setup

### PR DESCRIPTION
**Pull request by samgo1**
Travis CI initial script, related to issue #5

### CODE REVIEW

**Comment by samgo1** [comment]
the branch can be deleted once pull request is merged

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  components:
+    - build-tools-28.0.3
+    - android-28
+
+jobs:
+  include:
+    - stage: compile
+      language: android
+      before_script: chmod +x gradlew
+      script: ./gradlew assembleDebug assembleRelease
```
Comment (KrishnaPatel1) [inline .travis.yml]: I'd add 'clean' before 'assembleDebug' just for good measure

Comment (samgo1) [reply .travis.yml]: good suggestion, clean actually erases the build dir ( previous builds )

**Comment by aa-software2112** [comment]
This code will be useful for setting up the unit tests - later we may need to add to this an emulator in order to run application specific events - see https://github.com/aa-software2112/SOEN390_SimpleCamera/wiki/Testing-Environment for more details.

**Comment by aa-software2112** [comment]
I ran the test from scratch and the build holds for quite some time at this point 
image

Seems like it searches for the app-store signage and takes some time doing so... Any idea how we can save time with this?

**Comment by samgo1** [comment]
Related to issue #5

**Comment by samgo1** [comment]
> I ran the test from scratch and the build holds for quite some time at this point
> image
> 
> Seems like it searches for the app-store signage and takes some time doing so... Any idea how we can save time with this?

It doesn't search the app store it just prints out that we don't have a signing.properties file in the root directory of our app, the file of interest that builds our app is the build.gradle file located `/SOEN390_SimpleCamera/app/build.gradle` and is therefore responsible for the build time

**Comment by Lussier115** [comment]
We could create a temporary signing.properties file for the travis ci if its really needed.

**Comment by Lussier115** [comment]
Adding this may work, information in the file will probably be wrong and give this message: 'signing.properties found but some entries are missing'
We can find another way to fix it after if needed. 

before_script:
  - cp signing.properties_sample signing.properties

**Pull request merged**